common/tier4_autoware_utils/include/tier4_autoware_utils/ros/published_time_publisher.hpp

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Tier IV, Inc.
```

Comment:
This one might be better.
```suggestion
// Copyright 2024 The Autoware Contributors
```